php-packages/testing/src/integration/Extension/ExtensionManagerIncludeCurrent.php

```diff
@@ -112,4 +104,32 @@ protected function getAssetsFilesystem(): Cloud
     {
         return new FilesystemAdapter(new FlysystemFilesystem(new Local($this->paths->public.'/assets'), ['url' => resolve('flarum.config')->url().'/assets']));
     }
+
+    protected function includeCurrentExtension($package, string $packagePath)
+    {
+        if (Arr::get($package, 'type') === 'flarum-extension') {
+            $current = new Extension($packagePath, $package);
+            $current->setInstalled(true);
+            $current->setVersion(Arr::get($package, 'version'));
+            $current->calculateDependencies([], []);
```

Comment:
Might be cleaner if we redid all these with $withCalculatedDependencies$, $withVersion$, etc. Also, the function param here takes 1 arg, and we're giving it 2.